app/views/devise/sessions/new.html.erb

```diff
@@ -0,0 +1,26 @@
+<h2>Log in</h2>
```

Comment:
- [ ] OPTIONAL
This activity requires the modification of the views in devise for the registration and login to match the styling of the [provided wireframe(s)](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/sneak_peek.md). Kindly ensure that the form (inputs and buttons) stretches across the screen and is centered accordingly. Kindly ensure to improve upon the same

**Your login view:**
![image](https://user-images.githubusercontent.com/12745474/201043103-bb05f4a5-038d-4e21-b3d6-3396976b11dc.png)

**Wireframe login view:**
![image](https://user-images.githubusercontent.com/12745474/192667669-04020765-a648-4c50-ab45-7d68eda203e7.png)